src/stdlib/PairUtils.scillib

```diff
@@ -17,6 +17,7 @@
 (* ******************************************************************************** *)
 scilla_version 0
 
+import BoolUtils
```

Comment:
I guess this line is not needed anymore?

Reply:
BoolUtils is still needed for `andb` no?

https://github.com/Zilliqa/scilla/blob/5628c182e5b172f37d5bc9c0175eba0d16ea24e9/src/stdlib/CryptoUtils.scillib#L33

Reply:
yes, but this is `PairUtils.scillib`